### PR TITLE
Make variables org centric

### PR DESCRIPTION
**Pull request by ebb-tide**
Closes Variables part of #13063

### CODE REVIEW

File: ui/src/shared/components/ExportOverlay.tsx [hunk 1]
```diff
@@ -151,21 +142,14 @@ class ExportOverlay extends PureComponent<Props> {
...
-    const template = addOrgIDToTemplate(resource, orgID)
-
-    try {
-      await client.templates.create(template)
-      notify(resourceSavedAsTemplate(resourceName))
-    } catch (error) {
-      notify(saveResourceAsTemplateFailed(resourceName, error))
-    }
+    const {resource, onDismissOverlay, resourceName} = this.props
+    createTemplateFromResource(resource, resourceName)
```
Comment (ischolten) [inline ui/src/shared/components/ExportOverlay.tsx]: woot

File: ui/src/templates/actions/index.ts [hunk 2]
```diff
@@ -115,6 +114,22 @@ export const createTemplate = (template: DocumentCreate) => async dispatch => {
   }
 }
 
+export const createTemplateFromResource = (
```
Comment (ischolten) [inline ui/src/templates/actions/index.ts]: Noice!

File: ui/src/variables/actions/index.ts [hunk 3]
```diff
@@ -141,11 +145,16 @@ export const selectValue = (
...
+  getState: GetState
+) => {
   try {
     dispatch(setVariables(RemoteDataState.Loading))
-
-    const variables = await client.variables.getAll()
+    const {
+      orgs: {org},
+    } = getState()
+    const variables = (await client.variables.getAll(org.id)) as IVariable[]
```
Comment (ischolten) [inline ui/src/variables/actions/index.ts]: why do we need type coercion here? does get all not return it as the type we need?

File: ui/src/variables/components/VariableImportOverlay.tsx [hunk 4]
```diff
@@ -40,23 +32,19 @@ class VariableImportOverlay extends PureComponent<Props> {
...
-    orgID: string
+    uploadContent: string
   ): Promise<void> => {
-    const {createVariableFromTemplate, getVariables} = this.props
+    const {createVariableFromTemplate} = this.props
 
     const template = JSON.parse(uploadContent)
-    await createVariableFromTemplate(template, orgID)
-
-    await getVariables()
```
Comment (ischolten) [inline ui/src/variables/components/VariableImportOverlay.tsx]: is this no longer required to be able to see all imported variables?

Comment (ebb-tide) [reply ui/src/variables/components/VariableImportOverlay.tsx]: with variable templates including other variables, it's probably better to getVariables.

File: ui/src/variables/actions/index.ts (hunk 3 above)
Comment (ebb-tide) [reply ui/src/variables/actions/index.ts]: this needs to be fixed in client

**Pull request merged**